components/apimgt/org.wso2.carbon.apimgt.rest.api.store/src/test/java/org/wso2/carbon/apimgt/rest/api/store/impl/ExportApiServiceImplTestCase.java

```diff
@@ -0,0 +1,93 @@
+package org.wso2.carbon.apimgt.rest.api.store.impl;
```

Comment:
Lets Add License headers for test classes as well

Reply:
Done